src/buttons/buttonClose.command.js

```diff
@@ -8,15 +9,15 @@ export default class ButtonCloseCommands {
     }
 
     editor.runCommand('preset-mautic:dynamic-content-components-to-tokens');
-
     const htmlDocument = ContentService.getCanvasAsHtmlDocument(editor);
+    let htmlString = ContentService.serializeHtmlDocument(htmlDocument);
 
-    ButtonCloseCommands.returnContentToTextarea(
-      editor,
-      ContentService.serializeHtmlDocument(htmlDocument)
-    );
+    // eslint-disable-next-line no-undef
+    if (mauticEditorFonts) {
```

Comment:
the global `var mauticEditorFonts` can be added to [.eslintrc](https://github.com/mautic/grapesjs-preset-mautic/blob/main/.eslintrc) so the  `eslint-disable-next-line no-undef` is not needed. It has multiple occurrences.

Reply:
@adiux thank you, good idea! I added `mauticEditorFonts` to .eslintrc and removed unnecessary comments. Could you please check again?

Reply:
@adiux this is GTG if you can re-check and approve?

Reply:
Awesome! Marking this one as RTC - we will need to add this to the upcoming release notes for 4.4.9. cc @escopecz